sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/tests/Models/Employee.cs

```diff
@@ -15,7 +15,7 @@ namespace TestSchema
 	
 	public partial class Employee : ISpecificRecord
 	{
-		public static Schema _SCHEMA = Avro.Schema.Parse("{\"type\":\"record\",\"name\":\"Employee\",\"namespace\":\"TestSchema\",\"fields\":[{\"name\":\"Na" +
+		public static Schema _SCHEMA = Avro.Schema.Parse("{\"type\":\"record\",\"name\":\"Employee\",\"aliases\": [\"EmployeeV2\"],\"namespace\":\"TestSchema\",\"fields\":[{\"name\":\"Na" +
```

Comment:
It seems odd that you would need to put an alias for a later schema, but without this I would get mismatch exceptions due to the alias of the reader schema not matching the name of the writer schema.